care/emr/utils/file_manager.py

```diff
@@ -69,3 +73,27 @@ def file_contents(self, file_obj):
         content_type = response["ContentType"]
         content = response["Body"].read()
         return content_type, content
+
+    def delete_object(self, file_obj, **kwargs):
+        config, bucket_name = get_client_config(self.bucket_type)
+        s3 = boto3.client("s3", **config)
+
+        return s3.delete_object(
+            Bucket=bucket_name, Key=f"{file_obj.file_type}/{file_obj.internal_name}"
```

Comment:
What happens if the internal_name is `*` does it delete everything?

Reply:
Need to check that